web/themes/custom/westy/src/scss/components/_alerts.scss

```diff
@@ -18,3 +18,17 @@ header.page__header {
     }
   }
 }
+
+.global-alert-content {
+  p:last-child {
+    margin-bottom: 0;
+}
+  a {
+    color: $light;
+    text-decoration: underline;
+  }
+  .img {
```

Comment:
I don't think you need the .img as we don't have an SVG with that class outputting anywhere.